experimental/dds/tree/src/test/SharedTree.fuzz.tests.ts

```diff
@@ -0,0 +1,10 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { runSharedTreeFuzzTests } from './fuzz/SharedTreeFuzzTests';
```

Comment:
Depending on where we land on things, some of the changes in SharedTree could be reverted if we want. Specifically, I split out fuzz tests to a different file and inlined the `test:stress` command for two reasons:

1. The pipeline setup doesn't support `mocha -r node_modules/@fluidframework/mocha-test-setup` for some reason. The install tree is a bit wonky since we `npm i` the package we're testing and use it as the working directory to run tests. But honestly I'm not sure why we don't just use `mocha -r @fluidframework/mocha-test-setup` everywhere anyway--it seems simpler.
2. If we don't want to generically install all of the dev dependencies of the package we're testing, we need to be a bit more careful on which of those devDeps are runtime dependencies of the test command. This is less error-prone if the file is disjoint.

Reply:
There was some discussion on a teams thread about this topic which pointed out the reason for the `node_modules` in the include path is basically a workaround with the hoisting structure of the repo. Since these are all new commands, excluding should be ok anyway. But at least one mystery question answered :).